.travis.yml

```diff
@@ -13,26 +13,32 @@ env:
         - SETUP_CMD='test'
         - TEST_CMD='py.test test_env.py'
         - PYTHON_VERSION=3.6
+        - DEBUG=True
```

Comment:
If this is left as global here, we better add a few `DEBUG=False` ones to the various jobs below

Reply:
We can remove it once this is ready to merge